src/App.js

```diff
@@ -1,25 +1,61 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useEffect, useState} from "react";
+import Recipe from './Recipe';
+
+const App = () => {
+  const APP_ID = "6ada44de";
+  const APP_KEYS = "aa6e76fa436b639ff52a308980944f24";
+
+  const [recipes, setRecipes] = useState([]);
+  const [search, setSearch] = useState('');
+  const [query, setQuery] = useState('chicken');
+
+  useEffect( () => {
+    getRecipes();
+  }, [query]);
+
+  const getRecipes = async () => {
+    try {
+      const response = await fetch(`https://api.edamam.com/search?q=${query}&app_id=${APP_ID}&app_key=${APP_KEYS}&from=0&to=3&calories=591-722&health=alcohol-free`);
+      const data = await response.json();
+      setRecipes(data.hits);
+      console.log(data.hits);
+    } catch(e) {
+      setRecipes([]);
+      console.log(e);
```

Comment:
If you are actually catching an error you should throw the error rather than setting recipes to an arbitrary value. It would be something along the lines of `throw new Error(e)`